RELEASE.rst

```diff
@@ -0,0 +1,10 @@
+Release Instructions
+====================
+
+1. Prepare master branch for release (make sure all PRs are merged and tests pass).
+
+2. Run `bumpversion <major/minor/bugfix>`. This will create a new tagged commit,
```

Comment:
Is `bumpversion` a standard tool? We should probably have some info about where to install it from.

Reply:
It's on pypi, and it's in the requirements_dev.txt. Package name is bump2version, it behaves the same as the now unsupported bumpversion package.

Reply:
Okay sounds good, didn't realize it was in requirements_dev.txt